api/controller/psmdb_cluster.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package percona.platform.dbaas.controller.v1beta1;
+
+option go_package = "controller;controllerv1beta1";
+
+import "controller/common.proto";
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+
+// PSMDBClusterState represents PSMDB cluster CR state.
+enum PSMDBClusterState {
+  // PSMDB_CLUSTER_STATE_INVALID represents unknown state.
+  PSMDB_CLUSTER_STATE_INVALID = 0;
+  // PSMDB_CLUSTER_STATE_CHANGING represents a cluster being changed (initializing).
+  PSMDB_CLUSTER_STATE_CHANGING = 1;
+  // PSMDB_CLUSTER_STATE_READY represents a cluster without pending changes (ready).
+  PSMDB_CLUSTER_STATE_READY = 2;
+  // PSMDB_CLUSTER_STATE_FAILED represents a failed cluster (error).
```

Comment:
I vaguely remember that PXC and PSMDB operators used different states (those values in round brackets). Please check them

Reply:
there is one more state called `pending`, 
Should we use that?

Reply:
Please create a separate ticket for that in DBaaS foundation and assign to me, I will check when I'm back

Reply:
ok

Reply:
https://jira.percona.com/browse/PMM-6787